app/_src/gateway/breaking-changes/36x.md

```diff
@@ -88,11 +88,14 @@ The following is a list of known issues that may be fixed in a future release.
 
 ### Operating system requirements
 
-{{site.base_gateway}} 3.6.0.0 requires a higher [ulimit](https://linux.die.net/man/3/ulimit) to function properly. 
-It will not start properly with a ulimit set to 1024 or lower. We recommend setting the ulimit for your operating system to at least 4096. 
+{{site.base_gateway}} 3.6 requires a higher limit on the number of file descriptions to 
```

Comment:
```suggestion
{{site.base_gateway}} 3.6.0.0 requires a higher limit on the number of file descriptions to 
```
We should specifically call out the full version number here since the issue was fixed in 3.6.1.0.